source/grid/tria.cc

```diff
@@ -1747,9 +1747,11 @@ namespace internal
                 // throw an exception if no such cells should exist.
                 if (!triangulation.check_for_distorted_cells)
                   {
-                    const double cell_measure =
-                      GridTools::cell_measure<1>(triangulation.vertices,
-                                                 cells[cell_no].vertices);
+                    const double cell_measure = GridTools::cell_measure<1>(
+                      triangulation.vertices,
+                      reinterpret_cast<const unsigned int(
+                          &)[GeometryInfo<1>::vertices_per_cell]>(
+                        cells[cell_no].vertices.front()));
```

Comment:
Let's not do this. Just copy the vertices into an array of the right size, or introduce an overload of `GridTools::cell_measure`. 

Same in the other places.

Reply:
For example, you could introduce an overload that takes an `ArrayView` as argument. (Or replace the existing function -- that should work in the best of all cases without further changes.)

Reply:
I went with copying the values into a temporal array.